android-json-form-wizard/src/test/java/com/vijay/jsonwizard/customviews/NumericDatePickerTest.java

```diff
@@ -140,8 +140,6 @@ public void testNumericDatePickerSetMinValueLimitsCorrectly() {
         //Try set a date within min
         LocalDate dateFiveMonthsBeforeToday = localDate.minusMonths(6);
         datePicker.setDate(dateFiveMonthsBeforeToday.toDate());
-
-        Assert.assertEquals(localDate.getDayOfMonth(), datePicker.getDayOfMonth());
```

Comment:
Why are we removing the Assert here?

Reply:
Haven't been removed 
Removed spacing 
 Assert.assertEquals(dateFiveMonthsBeforeToday.getMonthOfYear() - 1, datePicker.getMonth());
        Assert.assertEquals(dateFiveMonthsBeforeToday.getYear(), datePicker.getYear());

Reply:
` Assert.assertEquals(localDate.getDayOfMonth(), datePicker.getDayOfMonth());` this is missing

Reply:
Done